components/Navigation/styles.js

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+import styled from "styled-components";
+import Icon from "@mdi/react";
+
+export const StyledNavBar = styled.div`
+  position: fixed;
+  bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 60px;
+  background-color: black;
+`;
+
+export const StyledNavBarLink = styled(Link)`
+  width: 25%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const StyledNavBarIcon = styled(Icon)`
+  color: ${({ isActive }) => (isActive ? "hotpink" : "whitesmoke")};
```

Comment:
`isActive` is spelled without a dollar sign, which is why you need to use the prop by the same name. Either include the dollar sign here or remove it everywhere else.